### PR TITLE
exercise-errors

### PR DESCRIPTION
**Pull request by nkoblai**

### CODE REVIEW

File: go-tour/errors.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+func (e ErrNegativeSqrt) Error() string {
+	return fmt.Sprint("Cannot Sqrt negative number: ", float64(e))
+}
+
+// Sqrt should return a non-nil error value when given a negative number, as it doesn't support complex numbers.
+func main() {
+	f, err := Sqrt(1)
+	if err != nil {
+		fmt.Println(err)
+	} else {
```
Comment (arbrix) [inline go-tour/errors.go]: `else` unnecessary in this case

File: go-tour/errors.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+// ErrNegativeSqrt returns non-nil error value when given a negative number.
+func (e ErrNegativeSqrt) Error() string {
+	return fmt.Sprint("Cannot Sqrt negative number: ", float64(e))
+}
+
+// Sqrt should return a non-nil error value when given a negative number, as it doesn't support complex numbers.
+func main() {
+	f, err := Sqrt(1)
```
Comment (arbrix) [inline go-tour/errors.go]: only one case, add more to validate error case

**Review by arbrix — CHANGES_REQUESTED** [review]

File: go-tour/errors.go (hunk 1 above)
Comment (nkoblai) [reply go-tour/errors.go]: added one more case

File: go-tour/errors.go (hunk 2 above)
Comment (nkoblai) [reply go-tour/errors.go]: added one more case

**Comment by nkoblai** [comment]
@arbrix fixed

**Review by arbrix — APPROVED** [review]
LGTM, try to use table tests in the future

**Pull request merged**